### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by joaotavora**
mister, please be so kind as to review this

### CODE REVIEW

**Comment by luismbo** [comment]
See https://github.com/luismbo/stefil/tree/pr1. Should we rebase?

**Comment by luismbo** [comment]
Also, added some TODOs as issues.

**Comment by joaotavora** [comment]
Rebased and integrated your changes, did a `push -f`, github was kinda smart, the new commits are shown above, only they have been added just now, not 8 days ago.

File: source/suite.lisp [hunk 1]
```diff
@@ -56,3 +71,165 @@
...
+                assertion-count failed-assertion-count unexpected-error-count expected-count))))
+      (setf *pretty-log-accumulated-assertion-count* (assertion-count-of results))
+      (setf *pretty-log-accumulated-failure-descriptions* (failure-descriptions-of results))
+      (values-list v-list))))
+
+(defun indented-format (level stream format-control &rest format-arguments)
+  (let ((line-prefix (make-string level :initial-element #\Space)))
+    (let ((output (format nil "~?~%" format-control format-arguments)))
+      (with-input-from-string (s output)
+	(loop for line = (read-line s nil nil) until (null line)
```
Comment (luismbo) [inline source/suite.lisp]: Meh. Funky indentation. Get rid of tabs coming from sc land. :-)

File: test/stefil-test.asd [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+;;; Copyright (c) 2009 by the authors.
+;;;
+;;; See LICENCE for details.
+
+(defsystem :stefil-test
+  :licence "BSD / Public domain"
+  :depends-on (:stefil)
+  :serial t
+  :components ((:file "package")
+	       (:file "basic")
```
Comment (luismbo) [inline test/stefil-test.asd]: More funky indentation?

Comment (joaotavora) [inline test/stefil-test.asd]: is funny. is like someone else visit sc-land and blame it on me! https://github.com/luismbo/stefil/commit/482b6f7b44119da02e26833e303cdaf3fb34f7cc 
Check your emacs `indent-tabs-mode` var.

Comment (luismbo) [inline test/stefil-test.asd]: My bad, I know.

On Wed, Apr 17, 2013 at 9:36 AM, João Távora notifications@github.comwrote:

> In test/stefil-test.asd:
> 
> > @@ -0,0 +1,13 @@
> > +;;; -_\- mode: Lisp; Syntax: Common-Lisp; -_-
> > +;;;
> > +;;; Copyright (c) 2009 by the authors.
> > +;;;
> > +;;; See LICENCE for details.
> > +
> > +(defsystem :stefil-test
> > -  :licence "BSD / Public domain"
> > -  :depends-on (:stefil)
> > -  :serial t
> > -  :components ((:file "package")
> > -          (:file "basic")
> 
> is funny. is like someone else visit sc-land and blame it on me! 482b6f7https://github.com/luismbo/stefil/commit/482b6f7b44119da02e26833e303cdaf3fb34f7cc
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/luismbo/stefil/pull/1/files#r3828535
> .

## 

Luís Oliveira
http://r42.eu/~luis/

**Pull request merged**